### PR TITLE
[DOCS-471] New team page

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Updates the Team section of DD doc to take into account the new design and features:

* Transform the team page into a full sub-section
* All user management is now at the root of this subsection
* Creation of a dedicated page for Datadog User Roles and Custom Roles

### Motivation
New feature is coming out

### Preview link

* [New Users sub-section](https://docs-staging.datadoghq.com/gus/new-team-page/account_management/users/)
* [Default Roles](https://docs-staging.datadoghq.com/gus/new-team-page/account_management/users/default_roles/)
* [Custom Roles](https://docs-staging.datadoghq.com/gus/new-team-page/account_management/users/custom_roles/)

### CODE REVIEW

File: content/en/account_management/team/roles.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+1. Go in your [Datadog Roles page][6].
+2. Select **New Role** in the upper right corner of the page.
+3. Give a name to your Role.
+4. Select the permission associated with this.
+
+{{< img src="account_management/team/create_role.png" alt="Create a custom Role" responsive="true" style="width:90%;">}}
+
+Once a role is created you can [add this role to existing users][7].
+
+### List of availables permissions
```
Comment (l0k0ms) [inline content/en/account_management/team/roles.md]:
```suggestion
### List of available permissions
```

File: content/en/account_management/team/_index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+further_reading:
+- link: "account_management/saml"
+  tag: "Documentation"
+  text: "Configure SAML for your Datadog account"
+- link: "account_management/multi_organization"
+  tag: "Documentation"
+  text: "Configuring Teams & Organizations with Multiple Accounts"
+---
+
+The Team section of Datadog allows you to manage your users and their associated Roles:
```
Comment (cswatt) [inline content/en/account_management/team/_index.md]:
```suggestion
Datadog's **Team** section allows you to manage your users and their associated roles:
```

File: content/en/account_management/team/_index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+1. Go to the [Team Page][1].
+2. Select **Invite Users** in the upper right corner of the page.
+3. Enter the email address of the user you wish to invite to your Datadog account.
+4. Assign a [user role][2] to this user.
+5. Click **Send Invites**
+
+{{< img src="account_management/team/invite_user.png" alt="Add Member To Team" responsive="true" style="width:80%;">}}
+
+The new user will receive an email with a link to log in. Until it's done, the user is marked as: status `Pending`.
```
Comment (cswatt) [inline content/en/account_management/team/_index.md]:
```suggestion
The new user will receive an email with a link to log in. The user is marked with status `Pending` until they log in.
```

File: content/en/account_management/team/_index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+To re-send an invite, select the *Edit* button on the right of the user line, then click *Resend Invite*:
+
+{{< img src="account_management/team/resend_invite.png" alt="Resend invite" responsive="true" style="width:80%;">}}
+
+## Edit a user roles
```
Comment (cswatt) [inline content/en/account_management/team/_index.md]:
```suggestion
## Edit a user's roles
```

File: content/en/account_management/team/_index.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Only team administrators can promote members:
+
+1. Go to the [Team Page][1].
+2. Select the *Edit* button on the right of the user line
```
Comment (cswatt) [inline content/en/account_management/team/_index.md]:
```suggestion
2. Select the *Edit* button on the right of the user line.
```

File: content/en/account_management/team/_index.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+3. Select the new [user roles][2] for this user
```
Comment (cswatt) [inline content/en/account_management/team/_index.md]:
```suggestion
3. Select the new [user roles][2] for this user.
```

File: content/en/account_management/team/_index.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+4. **Save** the new setting.
+
+{{< img src="account_management/team/user_role_update.png" alt="User role update" responsive="true" style="width:80%;">}}
+
+## Disable existing members
+
+You must be an administrator of the team to disable members. You cannot permanently remove users as they might own events, dashboards, etc. that are not supposed to be removed.
```
Comment (cswatt) [inline content/en/account_management/team/_index.md]:
```suggestion
Only administrators of the team can disable members. You cannot permanently remove users, as they might own events, dashboards, etc. that are not supposed to be removed.
```

File: content/en/account_management/team/_index.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+1. Go to the [Team Page][1].
+2. Select the *Edit* button on the right of the user line
+3. Click on the **Disable** toggle.
+4. **Save** the new setting.
+
+{{< img src="account_management/team/disable_user.png" alt="Disable User" responsive="true" style="width:80%;" >}}
+
+**Note**: Disabled team members disappear from the administrator's team page UI automatically after one month.
```
Comment (cswatt) [inline content/en/account_management/team/_index.md]:
```suggestion
**Note**: Disabled team members disappear from the administrator's team page automatically after one month.
```

File: content/en/account_management/team/roles.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+  text: "Manage your Datadog Users"
+- link: "account_management/saml"
+  tag: "Documentation"
+  text: "Configure SAML for your Datadog account"
+- link: "account_management/multi_organization"
+  tag: "Documentation"
+  text: "Configuring Teams & Organizations with Multiple Accounts"
+---
+
+Datadog provides 3 user roles by default:
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
Datadog provides three user roles by default:
```

File: content/en/account_management/team/roles.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Admin**     | Users have access to billing information and the ability to revoke API keys. They can manage users and configure [read-only dashboards][1]. They can also promote standard users to administrators.                                           |
+| **Standard**  | Users are allowed to view and modify all monitoring features that Datadog offers, such as [dashboards][1], [monitors][2], [events][3], and [notebooks][4]. Standard users can also invite other users to organizations.                       |
+| **Read Only** | Users do not have access to edit within Datadog. This comes in handy when you'd like to share specific read-only views with a client, or where a member of one business unit needs to share a [dashboard][5] with someone outside their unit. |
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
| **Read Only** | Users do not have access to edit within Datadog. This comes in handy when you'd like to share specific read-only views with a client, or when a member of one business unit needs to share a [dashboard][5] with someone outside their unit. |
```

File: content/en/account_management/team/roles.md [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+
+## Create a custom Role
+
+<div class="alert alert-warning">
+This feature is in private beta. <a href="/help">Contact Datadog support</a> to get it enabled for your account.
+</div>
+
+To create a custom Role:
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
To create a custom role:
```

File: content/en/account_management/team/roles.md [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Admin**     | Users have access to billing information and the ability to revoke API keys. They can manage users and configure [read-only dashboards][1]. They can also promote standard users to administrators.                                           |
+| **Standard**  | Users are allowed to view and modify all monitoring features that Datadog offers, such as [dashboards][1], [monitors][2], [events][3], and [notebooks][4]. Standard users can also invite other users to organizations.                       |
+| **Read Only** | Users do not have access to edit within Datadog. This comes in handy when you'd like to share specific read-only views with a client, or where a member of one business unit needs to share a [dashboard][5] with someone outside their unit. |
+
+## Create a custom Role
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
## Create a custom role
```

File: content/en/account_management/team/roles.md [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+
+<div class="alert alert-warning">
+This feature is in private beta. <a href="/help">Contact Datadog support</a> to get it enabled for your account.
+</div>
+
+To create a custom Role:
+
+1. Go in your [Datadog Roles page][6].
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
1. Go to your [Datadog Roles page][6].
```

File: content/en/account_management/team/roles.md [hunk 14]
```diff
@@ -0,0 +1,90 @@
...
+2. Select **New Role** in the upper right corner of the page.
+3. Give a name to your Role.
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
3. Give a name to your role.
```

File: content/en/account_management/team/roles.md [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+4. Select the permission associated with this.
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
4. Select the permissions associated with this role.
```

File: content/en/account_management/team/roles.md [hunk 16]
```diff
@@ -0,0 +1,90 @@
...
+
+{{< img src="account_management/team/create_role.png" alt="Create a custom Role" responsive="true" style="width:90%;">}}
+
+Once a role is created you can [add this role to existing users][7].
+
+### List of available permissions
+
+All permissions have up to 3 options that can be selected: Read, Write, and Other. Note that not all options are available every time for a given permission. Find below the detailed of those options and the impact over each available permission.
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
All permissions have up to three options that can be selected: Read, Write, and Other. Note that not all options are available every time for a given permission. Find below the detailed of those options and the impact of each available permission.
```

File: content/en/account_management/team/roles.md [hunk 17]
```diff
@@ -0,0 +1,90 @@
...
+| Log Processors         | *undefined*                         | Update the log processors in an index      | *undefined*                             |
+| Log External Archives  | *undefined*                         | Update the external archives configuration | *undefined*                             |
+| Logs Public Config API | *undefined*                         | *undefined*                                | Access the Logs Public Config API (r/w) |
+| Log Generate Metrics   | Access the Generate Metrics feature | *undefined*                                | *undefined*                             |
+
+#### Dashboards
+
+| Permission Name | Read                         | Write             | Other                     |
+|-----------------|------------------------------|-------------------|---------------------------|
+| Dashboards      | Access Dashboards (required) | Update Dashboards | Share Dashboards publicly |
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
| Dashboards      | Access dashboards (required) | Update dashboards | Share dashboards publicly |
```

File: content/en/account_management/team/roles.md [hunk 18]
```diff
@@ -0,0 +1,90 @@
...
+
+#### Monitors
+
+| Permission Name   | Read                       | Write                   | Other       |
+|-------------------|----------------------------|-------------------------|-------------|
+| Monitors          | Access Monitors (required) | Update monitor          | *undefined* |
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
| Monitors          | Access monitors (required) | Update monitor          | *undefined* |
```

File: content/en/account_management/team/roles.md [hunk 19]
```diff
@@ -0,0 +1,90 @@
...
+| Monitors Downtime | *undefined*                | Manage Monitor Downtime | *undefined* |
```
Comment (cswatt) [inline content/en/account_management/team/roles.md]:
```suggestion
| Monitors Downtime | *undefined*                | Manage monitor downtime | *undefined* |
```

**Comment by ruthnaebeck** [comment]
@l0k0ms - Checking in on this one

**Pull request merged**